pose-detection/src/calculators/keypoints_one_euro_filter.ts

```diff
@@ -47,28 +47,16 @@ export class KeypointsOneEuroFilter implements KeypointsFilter {
     // Initialize filters once.
     this.initializeFiltersIfEmpty(keypoints);
 
-    // Get value scale as inverse value of the object scale.
-    // If value is too small smoothing will be disabled and keypoints will be
-    // returned as is.
-    let valueScale = 1;
```

Comment:
Why is this logic removed? It's mandatory for BlazePose smoothing.
Same for all changes below.

Reply:
This change was temporary and is now gone in the newer commits